### ISSUE TITLE
Issue in lua-writer doc

### ISSUE THREAD

**Issue by benabel** [jgm/pandoc#9879]
Hello, I'm trying to write a custom Writer using the markdown template, but using the code snippet provided in the [doc](https://pandoc.org/custom-writers.html#example-modified-markdown-writer):

```lua
function Writer(doc, opts)
    local filter = {
        CodeBlock = function(cb)
            -- only modify if code block has no attributes
            if cb.attr == pandoc.Attr() then
                local delimited = '```\n' .. cb.text .. '\n```'
                return pandoc.RawBlock('markdown', delimited)
            end
        end
    }
    return pandoc.write(doc:walk(filter), 'gfm', opts)
end

function Template()
    local template = pandoc.template
    return template.compile(template.default 'gfm')
end
```

I get this **error**:

```
pandoc --standalone  --from markdown  --to utils/markdown_writer.lua '/home/ben/Documents/git/0lyceum/iut-info/BUT/R206-exp-bdd/2-sgbd/index.qmd' -o '/home/ben/Documents/git/0lyceum/iut-info/BUT/R206-exp-bdd/2-sgbd/index.qmd'
Error running Lua:
string expected, got pandoc Template
```

**Comment by tarleb** [comment]
Thanks for the report. This was a documentation bug that was fixed [here](f3ed3b36142afe3d005d57481d7b25c6f2db010a), but hasn't been deployed yet.

This should work:

``` lua
Template = pandoc.template.default 'gfm'
```

**Comment by benabel** [comment]
Thanks a lot @tarleb, `Template = pandoc.template.default 'gfm'` fix it.
